R/S3.R

```diff
@@ -66,6 +66,9 @@
 #'   have an R7 class inherit from an S3 class. It must be specified in the
 #'   same way as a R7 constructor: the first argument should be `.data`
 #'   (the base type whose attributes will be modified).
```

Comment:
I know you mention the no argument case below, but it may be helpful to explicitly advise here that `.data` should have a default value